.github/workflows/tox.yml

```diff
@@ -20,12 +20,12 @@ jobs:
 
     steps:
       - name: Check out the repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Set up Python 3.10
-        uses: actions/setup-python@v1
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
```

Comment:
It looks like using the latest GH action for Python setup is the key - so this will need changing in all of the repos (and platform too, for that matter).